lib/src/json_document_transformer.dart

```diff
@@ -22,7 +22,7 @@ final jsonDocument = new JsonDocumentTransformer();
 /// [FormatException]. If an unencodable object is added to the sink, it
 /// synchronously throws a [JsonUnsupportedObjectError].
 class JsonDocumentTransformer
-    implements StreamChannelTransformer<Object, String> {
+    implements StreamChannelTransformer<dynamic, String> {
```

Comment:
Same question as in the other PR: what's the difference between `Object` and `dynamic` here?